k8s-ci/k8s_crds/test

```diff
@@ -161,4 +166,4 @@ function main() {
   run_test_case "update configuration object" ohnoeswechangedit supersecretpassword
 }
 
-main
+main || exit_err "Failed inside main"
```

Comment:
Maybe we need something a bit more descriptive like the suggestion but feel free to change it.
```suggestion
main || exit_err "K8s-CRD test: Failed inside main()"
```